csharp-basic-topics/IntParseVsConvertToInt/IntParseVsConvertToInt/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿
+namespace IntParseVsConvertToInt
+{
+    public class Program
+    {
+        public static void IntParseMethod(string inputString)
+        {
+            try
+            {
+                int outputInteger = int.Parse(inputString);
```

Comment:
Use var instead of explicit types whenever possible. Apply to the entire source code.